### DISCUSSION TITLE
Make "updated at" info visible everytime in admin

### DISCUSSION THREAD

**Discussion by simi** [bensheldon/good_job#1109] [Ideas]
Currently update at info is visible at the bottom of the page and you need to scroll down to reach it. Usually I have good job dashboard opened with live poll and I would like to quickly check how fresh the dashboard it.

![image](https://github.com/bensheldon/good_job/assets/193936/5fdd4c6f-bed6-43d0-a113-dd9a03453a2c)

What about to move it somewhere it would be visible everytime like in the fixed header? It would be possble to move the "wording" at the same time and eliminate whole footer at the same time.

my initial idea

![image](https://github.com/bensheldon/good_job/assets/193936/33d0e7e0-36fc-4916-bf21-23e00db8aedd)

**Comment by bensheldon** [comment]
I like it 👍🏻 I guess the downside is that it uses a little more vertical space at the top, but we aren't optimizing anything else right now (me: staring at that giant "Jobs" heading element), so that should be fine.

Can you make that change? I think it could be another Bootstrap `nav` but I'm not sure about the margins and everything. I'm happy to tweak it if you open an initial PR.

**Reply by simi** [reply]
@bensheldon I'll open PR soon.

**Comment by simi** [comment]
partially fixed by https://github.com/bensheldon/good_job/pull/1204

**Reply by bensheldon** [reply]
What did we miss? 🤔
